skfda/datasets/_samples_generators.py

```diff
@@ -135,104 +255,296 @@ def euler_maruyama(  # noqa: WPS210
 
-        drift_function = constant_drift
+    An SDE can be expressed with the following formula
 
-    if diffusion is None:
-        if diffusion_matricial_term:
-            diffusion = np.eye(dim_codomain)
-        else:
-            diffusion = 1.0
+    .. math::
 
-    if callable(diffusion):
-        diffusion_function = diffusion
-    else:
-        def constant_diffusion(  # noqa: WPS430 -- We need internal functions
-            t: float,
-            x: NDArrayFloat,
-        ) -> NDArrayFloat:
-            return np.atleast_1d(diffusion)
+        d\mathbf{X}(t) = \mathbf{F}(t, \mathbf{X}(t))dt + \mathbf{G}(t,
+        \mathbf{X}(t))d\mathbf{W}(t).
 
-        diffusion_function = constant_diffusion
+    In this equation, :math:`\mathbf{X} = (X^{(1)}, X^{(2)}, ... , X^{(n)})
+    \in \mathbb{R}^q` is a vector that represents the state of the stochastic
+    process. The function :math:`\mathbf{F}(t, \mathbf{X}) = (F^{(1)}(t,
+    \mathbf{X}), ..., F^{(q)}(t, \mathbf{X}))` is called drift and refers
+    to the deterministic component of the equation. The function
+    :math:`\mathbf{G} (t, \mathbf{X}) = (G^{i, j}(t, \mathbf{X}))_{i=1, j=1}
+    ^{q, m}` is denoted as the diffusion term and refers to the stochastic
+    component of the evolution. :math:`\mathbf{W}(t)` refers to a Wiener
+    process (Standard Brownian motion) of dimension :math:`m`. Finally,
+    :math:`q` refers to the dimension of the variable :math:`\mathbf{X}`
+    (dimension of the codomain) and :math:`m` to the dimension of the noise.
+
+    Miltein's method computes the approximated solution using the
+    Markov chain
+
+    .. math::
+
+        X^{(i)}_{n + 1} = X_n^{(i)} + F^{(i)}(X_i, t_i)\Delta t_i +
+        \sum_{j=1}^m G^{i,j}(t_n, X_n)\sqrt{\Delta t_n}Z_n^j +
+        \sum_{j_1, j_2 = 1}^m L^{j_1} G^{i, j_2} (t_n, X_n)
+        I_{(j_1, j_2)} [t_n, t_{n+1}],
+
+    where :math:`X_n^{(i)}` is the approximated value of :math:`X^{(i)}(t_n)`
+    and the :math:`\mathbf{Z}_m` are independent, identically distributed
+    :math:`m`-dimensional standard normal random variables. :math:`L^j` stands
+    for the operator
+
+    .. math::
+
+        L^j = \sum_{k=1}^q G^{k, j}(t_n, X_n)\frac{\partial}{\partial x^k}
+
+
+    and :math:`I_{(j_1, j_2)} [t_n, t_{n+1}]` is the double stochastic integral
+
+    .. math::
 
-    def vector_diffusion_times_noise(  # noqa: WPS430 We need internal functons
+        I_{(j_1, j_2)} [t_n, t_{n+1}] = \int_{t_n}^{t_{n+1}} \int_{t}^{t_{n+1}}
+        dW_s^{j_1} dW_t^{j_2}.
+
+    In order to compute :math:`I_{(j_1, j_2)} [t_n, t_{n+1}]`, we use
+    Milstein L=0 method, described by Banerjee. This method approximates the
+    value of the double Itô integral by simulating solutions of another SDE.
+    The number of discretization points used to simulate the SDE which
+    approximates the double Itô integral is given by the parameter
+    *n_L0_discretization_points*.
```

Comment:
Parameters can be written between two pairs of backticks to show them in monospace font: \`\`param\`\`.